web/react/utils/client.jsx

```diff
@@ -849,3 +849,24 @@ module.exports.updateValetFeature = function(data, success, error) {
 
     module.exports.track('api', 'api_teams_update_valet_feature');
 };
+
+module.exports.isEmailEnabledSynchronous = function() {
+    var enabled = false;
+
+    $.ajax({
+        async: false,
```

Comment:
Oh that's unfortunate. Instead of making the server call when the user opens that dialog, I'll probably change it around so that it does it once when the page loads and then just caches the result. That'll be useful anyway when we fully remove the clientside settings file.